### PR TITLE
Iss 166

### LINKED ISSUE #166: Change GPS standard deviation threshold
`frame_calibration.cpp` currently has a minimum gps standard deviation set at 15 cm to filter out high covariance gps readings.

### PR DESCRIPTION
**Pull request by joeyzhu00**
closes #166

### CODE REVIEW

File: src/modules/localization/src/frame_tf_calibrate/frame_calibration.cpp [hunk 1]
```diff
@@ -88,7 +88,7 @@ namespace frame_calibration_node
         }
 
         // pitch multiplied by (-1) to adhere to ROS-105 frame standards
```
Comment (drofp) [inline src/modules/localization/src/frame_tf_calibrate/frame_calibration.cpp]:
```suggestion
        // in accordance to ROS-105 frame standards
```

File: src/utility/robot_localization/params/ekf_template.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ debug: false
 debug_out_file: /path/to/debug/file.txt
 
 # Whether to broadcast the transformation over the /tf topic. Defaults to true if unspecified.
-publish_tf: true
+publish_tf: false
```
Comment (drofp) [inline src/utility/robot_localization/params/ekf_template.yaml:39]: is there something else being used in place of this?

File: src/utility/robot_localization/params/ekf_template.yaml [hunk 3]
```diff
@@ -166,7 +167,7 @@ odom1_nodelay: false
 #                 false, false, false,
 #                 false, false, false] 
 # twist0_queue_size: 3
-# twist0_rejection_threshold: 2
+# twist0_rejection_hreshold: 2
```
Comment (drofp) [inline src/utility/robot_localization/params/ekf_template.yaml]:
```suggestion
# twist0_rejection_threshold: 2
```

**Review by drofp — CHANGES_REQUESTED** [review]

File: src/utility/robot_localization/params/ekf_template.yaml (hunk 2 above)
Comment (joeyzhu00) [reply src/utility/robot_localization/params/ekf_template.yaml:39]: Nothing is using this tf at the moment.

**Review by drofp — APPROVED** [review]

**Pull request merged**